mParticle-Taplytics/MPKitTaplytics.m

```diff
@@ -168,23 +169,24 @@ - (MPKitExecStatus *)setUserAttribute:(NSString *)key value:(NSString *)value {
     return [self createStatus:MPKitReturnCodeSuccess];
 }
 
-- (MPKitExecStatus *)setUserIdentity:(NSString *)identityString identityType:(MPUserIdentity)identityType {
-    MPKitReturnCode code;
-    switch( identityType ) {
-        case MPUserIdentityCustomerId: {
-            code = [[self setUserAttribute:@"user_id" value:identityString] returnCode];
-            break;
-        }
-        case MPUserIdentityEmail: {
-            code = [[self setUserAttribute:@"email" value:identityString] returnCode];
-            break;
-        }
-        default: {
-            code = MPKitReturnCodeUnavailable;
-            break;
-        }
-    }
-    return [self createStatus:code];
+- (void)setUserIdentitiesFromRequest:(FilteredMPIdentityApiRequest *)request {
+    NSDictionary *identities = [request userIdentities];
```

Comment:
just to make sure, `email` should be included in this as well if the `identity`/`login` call has `customerId` and `email`, correct? I'm waiting on my mParticle dashboard to update since I turned off email in `Connections > Platform > Users` and now waiting on it to propagate to my SDK.